core/src/main/java/org/commonjava/maven/ext/core/impl/DependencyManipulator.java

```diff
@@ -369,7 +372,7 @@ private void apply( final Project project, final Model model, final Map<Artifact
                 d.setGroupId( project.getModelParent().getGroupId() );
                 d.setArtifactId( project.getModelParent().getArtifactId() );
                 d.setVersion( project.getModelParent().getVersion() );
-                pDepMap.put( SimpleArtifactRef.parse( d.getManagementKey() ), d );
+                pDepMap.put( SimpleScopedArtifactRef.parse( d.getManagementKey() ), d );
```

Comment:
This is the reference to `SimpleScopedArtifactRef.parse( d.getManagementKey() )`.

Reply:
Great spot! Fixing.

Reply:
Looking now, I think this line is also what caused the Appveyor build failure.

Reply:
That would be me making a silly mistake - fixed now